package.json

```diff
@@ -3,13 +3,25 @@
   "version": "1.0.0",
   "description": "Simulate the browser's classList API for HAST nodes",
   "main": "index.js",
+  "type": "module",
```

Comment:
makes package [Pure ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)

maybe it needs to be noted in the README.md